app/src/main/java/org/mifos/mobile/ui/fragments/RecentTransactionsFragment.kt

```diff
@@ -38,17 +36,9 @@ import javax.inject.Inject
  */
 class RecentTransactionsFragment : BaseFragment(), RecentTransactionsView, OnRefreshListener {
 
-    @JvmField
-    @BindView(R.id.rv_recent_transactions)
-    var rvRecentTransactions: RecyclerView? = null
-
-    @JvmField
-    @BindView(R.id.swipe_transaction_container)
-    var swipeTransactionContainer: SwipeRefreshLayout? = null
-
-    @JvmField
-    @BindView(R.id.layout_error)
-    var layoutError: View? = null
+    private lateinit var errorLayoutBinding: ErrorLayoutBinding
```

Comment:
check the review left in ThirdPartyTransferFragment.kt regarding errorLayoutBinding and please make the necessary changes.